assets/eip-7885/README.md

```diff
@@ -0,0 +1,97 @@
+# NTT-EIP as a building block for FALCON, DILITHIUM and STARK verifiers
+
+This repository contains the EIP for NTT transform, along with a python reference code, and a solidity implementation.
```

Comment:
did you link this asset dir or this readme in the main EIP? if not recommend to do so for a user to navigate here